llvm/lib/Target/AMDGPU/GCNRegPressure.cpp

```diff
@@ -97,6 +97,46 @@ void GCNRegPressure::inc(unsigned Reg,
   Value[RegKind] += Sign;
 }
 
+struct RegExcess {
+  unsigned SGPR = 0;
+  unsigned VGPR = 0;
+  unsigned ArchVGPR = 0;
+  unsigned AGPR = 0;
```

Comment:
Not sure how to interpret splitting "ArchVGPR" and "AGPR". The hardware manual just usually spells out "AGPR" as ArchGPR so are these the same thing?